stl/inc/format

```diff
@@ -2953,15 +3052,54 @@ struct formatter<basic_string_view<_CharT, _Traits>, _CharT>
 
 template <class _CharT, class... _Args>
 struct _Basic_format_string {
+    static constexpr size_t _Num_args = sizeof...(_Args);
+
     basic_string_view<_CharT> _Str;
+    size_t _Arg_use_count[_Num_args > 0 ? _Num_args : 1]{};
+    size_t _Estimated_size    = 0;
+    bool _Is_estimation_exact = false;
 
     template <class _Ty>
         requires convertible_to<const _Ty&, basic_string_view<_CharT>>
     consteval _Basic_format_string(const _Ty& _Str_val) : _Str(_Str_val) {
         if (_Is_execution_charset_self_synchronizing()) {
+#ifndef __clang__ // TRANSITION, clang consteval bug (likely LLVM-52648)
+            _Format_checker<_CharT, remove_cvref_t<_Args>...> _Handler{_Str};
+            _Parse_format_string(_Str, _Handler);
+            _RANGES copy(_Handler._Arg_use_count, _Arg_use_count);
+            _Estimated_size      = _Handler._Estimated_size;
+            _Is_estimation_exact = _Handler._Is_estimation_exact;
+#else // ^^^ no workaround ^^^ / vvv workaround vvv
             _Parse_format_string(_Str, _Format_checker<_CharT, remove_cvref_t<_Args>...>{_Str});
+            _Estimated_size = _Str.size() + _Num_args * 8;

```

Comment:
This is not a workaround but a fallback to the other behavior

Reply:
I guess it's both a workaround and a fallback